scripts/SREP/mg-log-extractor/metadata.yaml

```
@@ -0,0 +1,31 @@
+file: script.sh
+name: mg-log-extractor
+description: |
+  Extracts must-gether logs from hive s3 bucket
```

Minor nits, non-blocking.
```suggestion
  Extracts must-gather logs from hive s3 bucket
```